artifacts/camara-cloudevents/event-subscription-template.yaml

```diff
@@ -447,20 +450,98 @@ components:
         - refreshToken
         - refreshTokenEndpoint
 
-    SubscriptionDetail:
-      description: The detail of the requested event subscription
-      type: object
-
     CreateSubscriptionDetail:
-      description: The detail of the requested event subscription
+      description: | 
```

Comment:
Think we should not add any specific model within this object and left empty, every project will customize the content of this object based on its scope.